arista-dcs-7124S/workflows/deploy-startup-config-graph.json

```diff
@@ -0,0 +1,31 @@
+{
+    "friendlyName": "Deploy Arista switch",
+    "injectableName": "Graph.Switch.Arista.Deploy",
+    "options": {
+        "deploy-config": {
+            "startupConfig": null,
+            "kickstartImage": null,
+            "bootImage": null
+        }
+    },
+    "tasks": [
+        {
+            "label": "deploy-startup-config",
+            "taskDefinition": {
+                "friendlyName": "Deploy switch config",
+                "injectableName": "Task.Inline.Switch.Deploy.Arista",
+                "implementsTask": "Task.Base.Linux.Commands",
+                "options": {
+                    "startupConfig": null,
+                    "startupConfigUri": "{{ api.base }}/templates/{{ options.startupConfig }}",
+                    "commands": [
+                        {
+                            "downloadUrl": "/api/1.1/templates/arista-deploy-startup-config.py"
```

Comment:
Oh, yeah I misread it. `{{ api.base }}` would still be good though, that way we can change the API version of the route internally.